pkg/server/commands/write_authzmodel_test.go

```diff
@@ -31,41 +29,21 @@ func TestWriteAuthorizationModelWithExperimentalEnableModularModels(t *testing.T
 	mockDatastore.EXPECT().MaxTypesPerAuthorizationModel().AnyTimes().Return(100)
 
 	testCases := map[string]struct {
-		enableModularModules bool
-		inputSchema          string
-		expectAllowed        bool
```

Comment:
Please rename `TestWriteAuthorizationModelWithExperimentalEnableModularModels` to `TestWriteAuthorizationModel`

Reply:
Doh, thanks for catching!